tests/Unit/UserServiceTest.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Services\UserService;
+use Tests\TestCase;
+
+class UserServiceTest extends TestCase
+{
+
+    public function test_create_user()
+    {
+        $this->artisan('migrate');
```

Comment:
why did you add `$this->artisan('migrate');` instead of the 'RefreshDatabase' trait as in the previous test?